csrc/python_frontend/fusion_record.h

```diff
@@ -1820,6 +1820,58 @@ struct IndexSelectOpRecord : RecordFunctor {
   int64_t dim_;
 };
 
+struct SelectOpRecord : RecordFunctor {
```

Comment:
nitpick. I wish we were a bit smarter and would have `IndexSelectopRecord` be general enough to be reused here.
But we don't have to worry about that for now. Can you please add a `TODO` here for merging indexing operations?